### ISSUE TITLE
XP tap issue in nether

### ISSUE THREAD

**Issue by sb555** [vadis365/Mob-Grinding-Utils#148]
When returning to a blaze grinder in the nether, nothing happens when you open the xp tap. You have to break it and place it down again, then it will work. This happens every time.

**Comment by Flanks255** [comment]
what minecraft version? cause i just tried in 1.16.5 and it worked fine...

**Comment by sb555** [comment]
Yes, 1.16.5. I have to travel quite a distance from the portal to reach it, and since I'm not using any chunkloaders with it, I'm wondering if it's an issue with the chunks being reloaded when getting near the tap again.
